uberduck_ml_dev/data_loader.py

```diff
@@ -240,37 +240,22 @@ def __call__(self, batch):
             f0_padded = torch.FloatTensor(len(batch), 1, max_target_len)
             f0_padded.zero_()
 
+        # pdb.set_trace()
         for i in range(len(ids_sorted_decreasing)):
             mel = batch[ids_sorted_decreasing[i]][1]
             mel_padded[i, :, : mel.size(1)] = mel
             gate_padded[i, mel.size(1) - 1 :] = 1
             output_lengths[i] = mel.size(1)
             speaker_ids[i] = batch[ids_sorted_decreasing[i]][2]
-            if self.include_f0:
-                f0 = batch[ids_sorted_decreasing[i]][3]
-                f0_padded[i, :, : f0.size(1)] = f0
-
-        # NOTE(zach): would model_inputs be better as a namedtuple or dataclass?
-        if self.include_f0:
```

Comment:
are we intending to remove the mellotron functionality? if so, should we just remove the whole mellotron model? Seems strange to remove the functionality from here but keep the rest of the functionality in this codebase. We should probably discuss this.

Reply:
the plan is to bring it back but it is not as urgent as having working tacotron2